examples/VLCB_4in4out/VLCB_4in4out.ino

```diff
@@ -112,10 +112,11 @@ const byte NUM_SWITCHES = sizeof(SWITCH) / sizeof(SWITCH[0]);
 // module objects
 Bounce moduleSwitch[NUM_SWITCHES];  //  switch as input
 LEDControl moduleLED[NUM_LEDS];     //  LED as output
-byte switchState[NUM_SWITCHES];
+bool state[NUM_SWITCHES];
 
 // forward function declarations
 void eventhandler(byte, const VLCB::VlcbMessage *);
+void requesthandler(byte, const VLCB::VlcbMessage *);
```

Comment:
This declaration is not needed. The function requesthandler() is not used anywhere.

Reply:
Sorry, I thought I'd done that.  In fact I had in my working copy but forgot to update Github.